pkg/utils/validator/clusterresourceoverride.go

```diff
@@ -79,3 +90,20 @@ func validateClusterResourceOverrideResourceLimit(cro fleetv1alpha1.ClusterResou
 	}
 	return errors.NewAggregate(allErr)
 }
+
+// validateClusterResourceOverrideRuleSelector checks if override rule is selecting resource by name.
+func validateClusterResourceOverrideRuleSelector(cro fleetv1alpha1.ClusterResourceOverride) error {
+	allErr := make([]error, 0)
+	for _, rule := range cro.Spec.Policy.OverrideRules {
+		if rule.ClusterSelector == nil {
+			continue
+		}
+		for _, selector := range rule.ClusterSelector.ClusterSelectorTerms {
+			// Check that only label selector is supported
+			if selector.LabelSelector == nil || selector.PropertySelector != nil || selector.PropertySorter != nil {
+				allErr = append(allErr, fmt.Errorf("label selector is only supported for resource selection %+v", selector))
+			}
```

Comment:
we also need to validate JSONPatchOverride to validate value https://github.com/Azure/fleet/blob/main/apis/placement/v1alpha1/override_types.go#L152

btw, the api will be changed in #748 and you can validate it in a separate PR once #748 is merged.

Reply:
Addressed in [#752 ](https://github.com/Azure/fleet/pull/752)